### ISSUE TITLE
with-next-page-transitions - Module parse failed: Bad escape sequence in untagged template literal (62:830)

### ISSUE THREAD

**Issue by umutozdemir97** [zeit/next.js#9065]
# Examples bug report

## Example name
next-page-transitions example

## Describe the bug

I downloaded the example with create-next-app, in the directory, I issued `yarn dev`. An error arised when I opened `localhost:3000`:

```
PS C:\Users\umuto\Desktop\with-next-page-transitions> yarn dev
yarn run v1.17.3
$ next
[ wait ]  starting the development server ...
[ info ]  waiting on http://localhost:3000 ...
[ error ] ./pages/_app.js 62:830
Module parse failed: Bad escape sequence in untagged template literal (62:830)
File was processed with these loaders:
 * ./node_modules/next/dist/build/webpack/loaders/next-babel-loader.js
You may need an additional loader to handle the result of these loaders.
|       id: "2096339533",
|       __self: this
>     }, `.page-transition-enter{opacity:0;-webkit-transform:translate3d(0,20px,0);-ms-transform:translate3d(0,20px,0);transform:translate3d(0,20px,0);}.page-transition-enter-active{opacity:1;-webkit-transform:translate3d(0,0,0);-ms-transform:translate3d(0,0,0);transform:translate3d(0,0,0);-webkit-transition:opacity ${TIMEOUT}ms,-webkit-transform ${TIMEOUT}ms;-webkit-transition:opacity ${TIMEOUT}ms,transform ${TIMEOUT}ms;transition:opacity ${TIMEOUT}ms,transform ${TIMEOUT}ms;}.page-transition-exit{opacity:1;}.page-transition-exit-active{opacity:0;-webkit-transition:opacity ${TIMEOUT}ms;transition:opacity ${TIMEOUT}ms;}.loading-indicator-appear,.loading-indicator-enter{opacity:0;}.loading-indicator-appear-active,.loading-indicator-enter-active{opacity:1;-webkit-transition:opacity ${TIMEOUT}ms;transition:opacity ${TIMEOUT}ms;}
| /*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkM6XFxVc2Vyc1xcdW11dG9cXERlc2t0b3BcXHdpdGgtbmV4dC1wYWdlLXRyYW5zaXRpb25zXFxwYWdlc1xcX2FwcC5qcyJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiQUFvQzJCLEFBR3VCLEFBSUEsQUFLQSxBQUdBLEFBS0EsQUFJQSxVQXBCd0IsQUFJSCxBQUtqQyxBQUdvRCxBQUtwRCxBQUlvRCxtR0FoQjBDLFNBSjlGLEFBWUEsQUFTQSxnU0FoQkEiLCJmaWxlIjoiQzpcXFVzZXJzXFx1bXV0b1xcRGVza3RvcFxcd2l0aC1uZXh0LXBhZ2UtdHJhbnNpdGlvbnNcXHBhZ2VzXFxfYXBwLmpzIiwic291cmNlc0NvbnRlbnQiOlsiaW1wb3J0IFJlYWN0IGZyb20gJ3JlYWN0J1xuaW1wb3J0IEFwcCBmcm9tICduZXh0L2FwcCdcbmltcG9ydCB7IFBhZ2VUcmFuc2l0aW9uIH0gZnJvbSAnbmV4dC1wYWdlLXRyYW5zaXRpb25zJ1xuXG5pbXBvcnQgTG9hZGVyIGZyb20gJy4uL2NvbXBvbmVudHMvTG9hZGVyJ1xuXG5jb25zdCBUSU1FT1VUID0gNDAwXG5cbmV4cG9ydCBkZWZhdWx0IGNsYXNzIE15QXBwIGV4dGVuZHMgQXBwIHtcbiAgc3RhdGljIGFzeW5jIGdldEluaXRpYWxQcm9wcyAoeyBDb21wb25lbnQsIGN0eCB9KSB7XG4gICAgbGV0IHBhZ2VQcm9wcyA9IHt9XG5cbiAgICBpZiAoQ29tcG9uZW50LmdldEluaXRpYWxQcm9wcykge1xuICAgICAgcGFnZVByb3BzID0gYXdhaXQgQ29tcG9uZW50LmdldEluaXRpYWxQcm9wcyhjdHgpXG4gICAgfVxuXG4gICAgcmV0dXJuIHsgcGFnZVByb3BzIH1cbiAgfVxuXG4gIHJlbmRlciAoKSB7XG4gICAgY29uc3QgeyBDb21wb25lbnQsIHBhZ2VQcm9wcyB9ID0gdGhpcy5wcm9wc1xuICAgIHJldHVybiAoXG4gICAgICA8PlxuICAgICAgICA8UGFnZVRyYW5zaXRpb25cbiAgICAgICAgICB0aW1lb3V0PXtUSU1FT1VUfVxuICAgICAgICAgIGNsYXNzTmFtZXM9J3BhZ2UtdHJhbnNpdGlvbidcbiAgICAgICAgICBsb2FkaW5nQ29tcG9uZW50PXs8TG9hZGVyIC8+fVxuICAgICAgICAgIGxvYWRpbmdEZWxheT17NTAwfVxuICAgICAgICAgIGxvYWRpbmdUaW1lb3V0PXt7XG4gICAgICAgICAgICBlbnRlcjogVElNRU9VVCxcbiAgICAgICAgICAgIGV4aXQ6IDBcbiAgICAgICAgICB9fVxuICAgICAgICAgIGxvYWRpbmdDbGFzc05hbWVzPSdsb2FkaW5nLWluZGljYXRvcidcbiAgICAgICAgPlxuICAgICAgICAgIDxDb21wb25lbnQgey4uLnBhZ2VQcm9wc30gLz5cbiAgICAgICAgPC9QYWdlVHJhbnNpdGlvbj5cbiAgICAgICAgPHN0eWxlIGpzeCBnbG9iYWw+e2BcbiAgICAgICAgICAucGFnZS10cmFuc2l0aW9uLWVudGVyIHtcbiAgICAgICAgICAgIG9wYWNpdHk6IDA7XG4gICAgICAgICAgICB0cmFuc2Zvcm06IHRyYW5zbGF0ZTNkKDAsIDIwcHgsIDApO1xuICAgICAgICAgIH1cbiAgICAgICAgICAucGFnZS10cmFuc2l0aW9uLWVudGVyLWFjdGl2ZSB7XG4gICAgICAgICAgICBvcGFjaXR5OiAxO1xuICAgICAgICAgICAgdHJhbnNmb3JtOiB0cmFuc2xhdGUzZCgwLCAwLCAwKTtcbiAgICAgICAgICAgIHRyYW5zaXRpb246IG9wYWNpdHkgJHtUSU1FT1VUfW1zLCB0cmFuc2Zvcm0gJHtUSU1FT1VUfW1zO1xuICAgICAgICAgIH1cbiAgICAgICAgICAucGFnZS10cmFuc2l0aW9uLWV4aXQge1xuICAgICAgICAgICAgb3BhY2l0eTogMTtcbiAgICAgICAgICB9XG4gICAgICAgICAgLnBhZ2UtdHJhbnNpdGlvbi1leGl0LWFjdGl2ZSB7XG4gICAgICAgICAgICBvcGFjaXR5OiAwO1xuICAgICAgICAgICAgdHJhbnNpdGlvbjogb3BhY2l0eSAke1RJTUVPVVR9bXM7XG4gICAgICAgICAgfVxuICAgICAgICAgIC5sb2FkaW5nLWluZGljYXRvci1hcHBlYXIsXG4gICAgICAgICAgLmxvYWRpbmctaW5kaWNhdG9yLWVudGVyIHtcbiAgICAgICAgICAgIG9wYWNpdHk6IDA7XG4gICAgICAgICAgfVxuICAgICAgICAgIC5sb2FkaW5nLWluZGljYXRvci1hcHBlYXItYWN0aXZlLFxuICAgICAgICAgIC5sb2FkaW5nLWluZGljYXRvci1lbnRlci1hY3RpdmUge1xuICAgICAgICAgICAgb3BhY2l0eTogMTtcbiAgICAgICAgICAgIHRyYW5zaXRpb246IG9wYWNpdHkgJHtUSU1FT1VUfW1zO1xuICAgICAgICAgIH1cbiAgICAgICAgYH08L3N0eWxlPlxuICAgICAgPC8+XG4gICAgKVxuICB9XG59XG4iXX0= */
| /*@ sourceURL=C:\Users\umuto\Desktop\with-next-page-transitions\pages\_app.js */`));
[ wait ]  compiling ...
[ error ] ./pages/_app.js 62:830
Module parse failed: Bad escape sequence in untagged template literal (62:830)
File was processed with these loaders:
 * ./node_modules/next/dist/build/webpack/loaders/next-babel-loader.js
You may need an additional loader to handle the result of these loaders.
|       id: "2096339533",
|       __self: this
>     }, `.page-transition-enter{opacity:0;-webkit-transform:translate3d(0,20px,0);-ms-transform:translate3d(0,20px,0);transform:translate3d(0,20px,0);}.page-transition-enter-active{opacity:1;-webkit-transform:translate3d(0,0,0);-ms-transform:translate3d(0,0,0);transform:translate3d(0,0,0);-webkit-transition:opacity ${TIMEOUT}ms,-webkit-transform ${TIMEOUT}ms;-webkit-transition:opacity ${TIMEOUT}ms,transform ${TIMEOUT}ms;transition:opacity ${TIMEOUT}ms,transform ${TIMEOUT}ms;}.page-transition-exit{opacity:1;}.page-transition-exit-active{opacity:0;-webkit-transition:opacity ${TIMEOUT}ms;transition:opacity ${TIMEOUT}ms;}.loading-indicator-appear,.loading-indicator-enter{opacity:0;}.loading-indicator-appear-active,.loading-indicator-enter-active{opacity:1;-webkit-transition:opacity ${TIMEOUT}ms;transition:opacity ${TIMEOUT}ms;}
| /*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkM6XFxVc2Vyc1xcdW11dG9cXERlc2t0b3BcXHdpdGgtbmV4dC1wYWdlLXRyYW5zaXRpb25zXFxwYWdlc1xcX2FwcC5qcyJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiQUFvQzJCLEFBR3VCLEFBSUEsQUFLQSxBQUdBLEFBS0EsQUFJQSxVQXBCd0IsQUFJSCxBQUtqQyxBQUdvRCxBQUtwRCxBQUlvRCxtR0FoQjBDLFNBSjlGLEFBWUEsQUFTQSxnU0FoQkEiLCJmaWxlIjoiQzpcXFVzZXJzXFx1bXV0b1xcRGVza3RvcFxcd2l0aC1uZXh0LXBhZ2UtdHJhbnNpdGlvbnNcXHBhZ2VzXFxfYXBwLmpzIiwic291cmNlc0NvbnRlbnQiOlsiaW1wb3J0IFJlYWN0IGZyb20gJ3JlYWN0J1xuaW1wb3J0IEFwcCBmcm9tICduZXh0L2FwcCdcbmltcG9ydCB7IFBhZ2VUcmFuc2l0aW9uIH0gZnJvbSAnbmV4dC1wYWdlLXRyYW5zaXRpb25zJ1xuXG5pbXBvcnQgTG9hZGVyIGZyb20gJy4uL2NvbXBvbmVudHMvTG9hZGVyJ1xuXG5jb25zdCBUSU1FT1VUID0gNDAwXG5cbmV4cG9ydCBkZWZhdWx0IGNsYXNzIE15QXBwIGV4dGVuZHMgQXBwIHtcbiAgc3RhdGljIGFzeW5jIGdldEluaXRpYWxQcm9wcyAoeyBDb21wb25lbnQsIGN0eCB9KSB7XG4gICAgbGV0IHBhZ2VQcm9wcyA9IHt9XG5cbiAgICBpZiAoQ29tcG9uZW50LmdldEluaXRpYWxQcm9wcykge1xuICAgICAgcGFnZVByb3BzID0gYXdhaXQgQ29tcG9uZW50LmdldEluaXRpYWxQcm9wcyhjdHgpXG4gICAgfVxuXG4gICAgcmV0dXJuIHsgcGFnZVByb3BzIH1cbiAgfVxuXG4gIHJlbmRlciAoKSB7XG4gICAgY29uc3QgeyBDb21wb25lbnQsIHBhZ2VQcm9wcyB9ID0gdGhpcy5wcm9wc1xuICAgIHJldHVybiAoXG4gICAgICA8PlxuICAgICAgICA8UGFnZVRyYW5zaXRpb25cbiAgICAgICAgICB0aW1lb3V0PXtUSU1FT1VUfVxuICAgICAgICAgIGNsYXNzTmFtZXM9J3BhZ2UtdHJhbnNpdGlvbidcbiAgICAgICAgICBsb2FkaW5nQ29tcG9uZW50PXs8TG9hZGVyIC8+fVxuICAgICAgICAgIGxvYWRpbmdEZWxheT17NTAwfVxuICAgICAgICAgIGxvYWRpbmdUaW1lb3V0PXt7XG4gICAgICAgICAgICBlbnRlcjogVElNRU9VVCxcbiAgICAgICAgICAgIGV4aXQ6IDBcbiAgICAgICAgICB9fVxuICAgICAgICAgIGxvYWRpbmdDbGFzc05hbWVzPSdsb2FkaW5nLWluZGljYXRvcidcbiAgICAgICAgPlxuICAgICAgICAgIDxDb21wb25lbnQgey4uLnBhZ2VQcm9wc30gLz5cbiAgICAgICAgPC9QYWdlVHJhbnNpdGlvbj5cbiAgICAgICAgPHN0eWxlIGpzeCBnbG9iYWw+e2BcbiAgICAgICAgICAucGFnZS10cmFuc2l0aW9uLWVudGVyIHtcbiAgICAgICAgICAgIG9wYWNpdHk6IDA7XG4gICAgICAgICAgICB0cmFuc2Zvcm06IHRyYW5zbGF0ZTNkKDAsIDIwcHgsIDApO1xuICAgICAgICAgIH1cbiAgICAgICAgICAucGFnZS10cmFuc2l0aW9uLWVudGVyLWFjdGl2ZSB7XG4gICAgICAgICAgICBvcGFjaXR5OiAxO1xuICAgICAgICAgICAgdHJhbnNmb3JtOiB0cmFuc2xhdGUzZCgwLCAwLCAwKTtcbiAgICAgICAgICAgIHRyYW5zaXRpb246IG9wYWNpdHkgJHtUSU1FT1VUfW1zLCB0cmFuc2Zvcm0gJHtUSU1FT1VUfW1zO1xuICAgICAgICAgIH1cbiAgICAgICAgICAucGFnZS10cmFuc2l0aW9uLWV4aXQge1xuICAgICAgICAgICAgb3BhY2l0eTogMTtcbiAgICAgICAgICB9XG4gICAgICAgICAgLnBhZ2UtdHJhbnNpdGlvbi1leGl0LWFjdGl2ZSB7XG4gICAgICAgICAgICBvcGFjaXR5OiAwO1xuICAgICAgICAgICAgdHJhbnNpdGlvbjogb3BhY2l0eSAke1RJTUVPVVR9bXM7XG4gICAgICAgICAgfVxuICAgICAgICAgIC5sb2FkaW5nLWluZGljYXRvci1hcHBlYXIsXG4gICAgICAgICAgLmxvYWRpbmctaW5kaWNhdG9yLWVudGVyIHtcbiAgICAgICAgICAgIG9wYWNpdHk6IDA7XG4gICAgICAgICAgfVxuICAgICAgICAgIC5sb2FkaW5nLWluZGljYXRvci1hcHBlYXItYWN0aXZlLFxuICAgICAgICAgIC5sb2FkaW5nLWluZGljYXRvci1lbnRlci1hY3RpdmUge1xuICAgICAgICAgICAgb3BhY2l0eTogMTtcbiAgICAgICAgICAgIHRyYW5zaXRpb246IG9wYWNpdHkgJHtUSU1FT1VUfW1zO1xuICAgICAgICAgIH1cbiAgICAgICAgYH08L3N0eWxlPlxuICAgICAgPC8+XG4gICAgKVxuICB9XG59XG4iXX0= */
| /*@ sourceURL=C:\Users\umuto\Desktop\with-next-page-transitions\pages\_app.js */`));
[ event ] build page: /
[ wait ]  compiling ...
[ error ] ./pages/_app.js 62:830
Module parse failed: Bad escape sequence in untagged template literal (62:830)
File was processed with these loaders:
 * ./node_modules/next/dist/build/webpack/loaders/next-babel-loader.js
You may need an additional loader to handle the result of these loaders.
|       id: "2096339533",
|       __self: this
>     }, `.page-transition-enter{opacity:0;-webkit-transform:translate3d(0,20px,0);-ms-transform:translate3d(0,20px,0);transform:translate3d(0,20px,0);}.page-transition-enter-active{opacity:1;-webkit-transform:translate3d(0,0,0);-ms-transform:translate3d(0,0,0);transform:translate3d(0,0,0);-webkit-transition:opacity ${TIMEOUT}ms,-webkit-transform ${TIMEOUT}ms;-webkit-transition:opacity ${TIMEOUT}ms,transform ${TIMEOUT}ms;transition:opacity ${TIMEOUT}ms,transform ${TIMEOUT}ms;}.page-transition-exit{opacity:1;}.page-transition-exit-active{opacity:0;-webkit-transition:opacity ${TIMEOUT}ms;transition:opacity ${TIMEOUT}ms;}.loading-indicator-appear,.loading-indicator-enter{opacity:0;}.loading-indicator-appear-active,.loading-indicator-enter-active{opacity:1;-webkit-transition:opacity ${TIMEOUT}ms;transition:opacity ${TIMEOUT}ms;}
| /*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkM6XFxVc2Vyc1xcdW11dG9cXERlc2t0b3BcXHdpdGgtbmV4dC1wYWdlLXRyYW5zaXRpb25zXFxwYWdlc1xcX2FwcC5qcyJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiQUFvQzJCLEFBR3VCLEFBSUEsQUFLQSxBQUdBLEFBS0EsQUFJQSxVQXBCd0IsQUFJSCxBQUtqQyxBQUdvRCxBQUtwRCxBQUlvRCxtR0FoQjBDLFNBSjlGLEFBWUEsQUFTQSxnU0FoQkEiLCJmaWxlIjoiQzpcXFVzZXJzXFx1bXV0b1xcRGVza3RvcFxcd2l0aC1uZXh0LXBhZ2UtdHJhbnNpdGlvbnNcXHBhZ2VzXFxfYXBwLmpzIiwic291cmNlc0NvbnRlbnQiOlsiaW1wb3J0IFJlYWN0IGZyb20gJ3JlYWN0J1xuaW1wb3J0IEFwcCBmcm9tICduZXh0L2FwcCdcbmltcG9ydCB7IFBhZ2VUcmFuc2l0aW9uIH0gZnJvbSAnbmV4dC1wYWdlLXRyYW5zaXRpb25zJ1xuXG5pbXBvcnQgTG9hZGVyIGZyb20gJy4uL2NvbXBvbmVudHMvTG9hZGVyJ1xuXG5jb25zdCBUSU1FT1VUID0gNDAwXG5cbmV4cG9ydCBkZWZhdWx0IGNsYXNzIE15QXBwIGV4dGVuZHMgQXBwIHtcbiAgc3RhdGljIGFzeW5jIGdldEluaXRpYWxQcm9wcyAoeyBDb21wb25lbnQsIGN0eCB9KSB7XG4gICAgbGV0IHBhZ2VQcm9wcyA9IHt9XG5cbiAgICBpZiAoQ29tcG9uZW50LmdldEluaXRpYWxQcm9wcykge1xuICAgICAgcGFnZVByb3BzID0gYXdhaXQgQ29tcG9uZW50LmdldEluaXRpYWxQcm9wcyhjdHgpXG4gICAgfVxuXG4gICAgcmV0dXJuIHsgcGFnZVByb3BzIH1cbiAgfVxuXG4gIHJlbmRlciAoKSB7XG4gICAgY29uc3QgeyBDb21wb25lbnQsIHBhZ2VQcm9wcyB9ID0gdGhpcy5wcm9wc1xuICAgIHJldHVybiAoXG4gICAgICA8PlxuICAgICAgICA8UGFnZVRyYW5zaXRpb25cbiAgICAgICAgICB0aW1lb3V0PXtUSU1FT1VUfVxuICAgICAgICAgIGNsYXNzTmFtZXM9J3BhZ2UtdHJhbnNpdGlvbidcbiAgICAgICAgICBsb2FkaW5nQ29tcG9uZW50PXs8TG9hZGVyIC8+fVxuICAgICAgICAgIGxvYWRpbmdEZWxheT17NTAwfVxuICAgICAgICAgIGxvYWRpbmdUaW1lb3V0PXt7XG4gICAgICAgICAgICBlbnRlcjogVElNRU9VVCxcbiAgICAgICAgICAgIGV4aXQ6IDBcbiAgICAgICAgICB9fVxuICAgICAgICAgIGxvYWRpbmdDbGFzc05hbWVzPSdsb2FkaW5nLWluZGljYXRvcidcbiAgICAgICAgPlxuICAgICAgICAgIDxDb21wb25lbnQgey4uLnBhZ2VQcm9wc30gLz5cbiAgICAgICAgPC9QYWdlVHJhbnNpdGlvbj5cbiAgICAgICAgPHN0eWxlIGpzeCBnbG9iYWw+e2BcbiAgICAgICAgICAucGFnZS10cmFuc2l0aW9uLWVudGVyIHtcbiAgICAgICAgICAgIG9wYWNpdHk6IDA7XG4gICAgICAgICAgICB0cmFuc2Zvcm06IHRyYW5zbGF0ZTNkKDAsIDIwcHgsIDApO1xuICAgICAgICAgIH1cbiAgICAgICAgICAucGFnZS10cmFuc2l0aW9uLWVudGVyLWFjdGl2ZSB7XG4gICAgICAgICAgICBvcGFjaXR5OiAxO1xuICAgICAgICAgICAgdHJhbnNmb3JtOiB0cmFuc2xhdGUzZCgwLCAwLCAwKTtcbiAgICAgICAgICAgIHRyYW5zaXRpb246IG9wYWNpdHkgJHtUSU1FT1VUfW1zLCB0cmFuc2Zvcm0gJHtUSU1FT1VUfW1zO1xuICAgICAgICAgIH1cbiAgICAgICAgICAucGFnZS10cmFuc2l0aW9uLWV4aXQge1xuICAgICAgICAgICAgb3BhY2l0eTogMTtcbiAgICAgICAgICB9XG4gICAgICAgICAgLnBhZ2UtdHJhbnNpdGlvbi1leGl0LWFjdGl2ZSB7XG4gICAgICAgICAgICBvcGFjaXR5OiAwO1xuICAgICAgICAgICAgdHJhbnNpdGlvbjogb3BhY2l0eSAke1RJTUVPVVR9bXM7XG4gICAgICAgICAgfVxuICAgICAgICAgIC5sb2FkaW5nLWluZGljYXRvci1hcHBlYXIsXG4gICAgICAgICAgLmxvYWRpbmctaW5kaWNhdG9yLWVudGVyIHtcbiAgICAgICAgICAgIG9wYWNpdHk6IDA7XG4gICAgICAgICAgfVxuICAgICAgICAgIC5sb2FkaW5nLWluZGljYXRvci1hcHBlYXItYWN0aXZlLFxuICAgICAgICAgIC5sb2FkaW5nLWluZGljYXRvci1lbnRlci1hY3RpdmUge1xuICAgICAgICAgICAgb3BhY2l0eTogMTtcbiAgICAgICAgICAgIHRyYW5zaXRpb246IG9wYWNpdHkgJHtUSU1FT1VUfW1zO1xuICAgICAgICAgIH1cbiAgICAgICAgYH08L3N0eWxlPlxuICAgICAgPC8+XG4gICAgKVxuICB9XG59XG4iXX0= */
| /*@ sourceURL=C:\Users\umuto\Desktop\with-next-page-transitions\pages\_app.js */`));
{ Error: Cannot find module 'C:\Users\umuto\Desktop\with-next-page-transitions\.next\server\static\development\pages\_document'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.loadComponents (C:\Users\umuto\Desktop\with-next-page-transitions\node_modules\next\dist\next-server\server\load-components.js:21:25)
    at DevServer.findPageComponents (C:\Users\umuto\Desktop\with-next-page-transitions\node_modules\next\dist\next-server\server\next-server.js:335:40)
    at DevServer.renderToHTML (C:\Users\umuto\Desktop\with-next-page-transitions\node_modules\next\dist\next-server\server\next-server.js:423:21)
    at DevServer.renderToHTML (C:\Users\umuto\Desktop\with-next-page-transitions\node_modules\next\dist\server\next-dev-server.js:17:534)
    at process._tickCallback (internal/process/next_tick.js:68:7) code: 'MODULE_NOT_FOUND' }
[ event ] build page: /next/dist/pages/_error
[ wait ]  compiling ...
[ error ] ./pages/_app.js 62:830
Module parse failed: Bad escape sequence in untagged template literal (62:830)
File was processed with these loaders:
 * ./node_modules/next/dist/build/webpack/loaders/next-babel-loader.js
You may need an additional loader to handle the result of these loaders.
|       id: "2096339533",
|       __self: this
>     }, `.page-transition-enter{opacity:0;-webkit-transform:translate3d(0,20px,0);-ms-transform:translate3d(0,20px,0);transform:translate3d(0,20px,0);}.page-transition-enter-active{opacity:1;-webkit-transform:translate3d(0,0,0);-ms-transform:translate3d(0,0,0);transform:translate3d(0,0,0);-webkit-transition:opacity ${TIMEOUT}ms,-webkit-transform ${TIMEOUT}ms;-webkit-transition:opacity ${TIMEOUT}ms,transform ${TIMEOUT}ms;transition:opacity ${TIMEOUT}ms,transform ${TIMEOUT}ms;}.page-transition-exit{opacity:1;}.page-transition-exit-active{opacity:0;-webkit-transition:opacity ${TIMEOUT}ms;transition:opacity ${TIMEOUT}ms;}.loading-indicator-appear,.loading-indicator-enter{opacity:0;}.loading-indicator-appear-active,.loading-indicator-enter-active{opacity:1;-webkit-transition:opacity ${TIMEOUT}ms;transition:opacity ${TIMEOUT}ms;}
| /*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkM6XFxVc2Vyc1xcdW11dG9cXERlc2t0b3BcXHdpdGgtbmV4dC1wYWdlLXRyYW5zaXRpb25zXFxwYWdlc1xcX2FwcC5qcyJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiQUFvQzJCLEFBR3VCLEFBSUEsQUFLQSxBQUdBLEFBS0EsQUFJQSxVQXBCd0IsQUFJSCxBQUtqQyxBQUdvRCxBQUtwRCxBQUlvRCxtR0FoQjBDLFNBSjlGLEFBWUEsQUFTQSxnU0FoQkEiLCJmaWxlIjoiQzpcXFVzZXJzXFx1bXV0b1xcRGVza3RvcFxcd2l0aC1uZXh0LXBhZ2UtdHJhbnNpdGlvbnNcXHBhZ2VzXFxfYXBwLmpzIiwic291cmNlc0NvbnRlbnQiOlsiaW1wb3J0IFJlYWN0IGZyb20gJ3JlYWN0J1xuaW1wb3J0IEFwcCBmcm9tICduZXh0L2FwcCdcbmltcG9ydCB7IFBhZ2VUcmFuc2l0aW9uIH0gZnJvbSAnbmV4dC1wYWdlLXRyYW5zaXRpb25zJ1xuXG5pbXBvcnQgTG9hZGVyIGZyb20gJy4uL2NvbXBvbmVudHMvTG9hZGVyJ1xuXG5jb25zdCBUSU1FT1VUID0gNDAwXG5cbmV4cG9ydCBkZWZhdWx0IGNsYXNzIE15QXBwIGV4dGVuZHMgQXBwIHtcbiAgc3RhdGljIGFzeW5jIGdldEluaXRpYWxQcm9wcyAoeyBDb21wb25lbnQsIGN0eCB9KSB7XG4gICAgbGV0IHBhZ2VQcm9wcyA9IHt9XG5cbiAgICBpZiAoQ29tcG9uZW50LmdldEluaXRpYWxQcm9wcykge1xuICAgICAgcGFnZVByb3BzID0gYXdhaXQgQ29tcG9uZW50LmdldEluaXRpYWxQcm9wcyhjdHgpXG4gICAgfVxuXG4gICAgcmV0dXJuIHsgcGFnZVByb3BzIH1cbiAgfVxuXG4gIHJlbmRlciAoKSB7XG4gICAgY29uc3QgeyBDb21wb25lbnQsIHBhZ2VQcm9wcyB9ID0gdGhpcy5wcm9wc1xuICAgIHJldHVybiAoXG4gICAgICA8PlxuICAgICAgICA8UGFnZVRyYW5zaXRpb25cbiAgICAgICAgICB0aW1lb3V0PXtUSU1FT1VUfVxuICAgICAgICAgIGNsYXNzTmFtZXM9J3BhZ2UtdHJhbnNpdGlvbidcbiAgICAgICAgICBsb2FkaW5nQ29tcG9uZW50PXs8TG9hZGVyIC8+fVxuICAgICAgICAgIGxvYWRpbmdEZWxheT17NTAwfVxuICAgICAgICAgIGxvYWRpbmdUaW1lb3V0PXt7XG4gICAgICAgICAgICBlbnRlcjogVElNRU9VVCxcbiAgICAgICAgICAgIGV4aXQ6IDBcbiAgICAgICAgICB9fVxuICAgICAgICAgIGxvYWRpbmdDbGFzc05hbWVzPSdsb2FkaW5nLWluZGljYXRvcidcbiAgICAgICAgPlxuICAgICAgICAgIDxDb21wb25lbnQgey4uLnBhZ2VQcm9wc30gLz5cbiAgICAgICAgPC9QYWdlVHJhbnNpdGlvbj5cbiAgICAgICAgPHN0eWxlIGpzeCBnbG9iYWw+e2BcbiAgICAgICAgICAucGFnZS10cmFuc2l0aW9uLWVudGVyIHtcbiAgICAgICAgICAgIG9wYWNpdHk6IDA7XG4gICAgICAgICAgICB0cmFuc2Zvcm06IHRyYW5zbGF0ZTNkKDAsIDIwcHgsIDApO1xuICAgICAgICAgIH1cbiAgICAgICAgICAucGFnZS10cmFuc2l0aW9uLWVudGVyLWFjdGl2ZSB7XG4gICAgICAgICAgICBvcGFjaXR5OiAxO1xuICAgICAgICAgICAgdHJhbnNmb3JtOiB0cmFuc2xhdGUzZCgwLCAwLCAwKTtcbiAgICAgICAgICAgIHRyYW5zaXRpb246IG9wYWNpdHkgJHtUSU1FT1VUfW1zLCB0cmFuc2Zvcm0gJHtUSU1FT1VUfW1zO1xuICAgICAgICAgIH1cbiAgICAgICAgICAucGFnZS10cmFuc2l0aW9uLWV4aXQge1xuICAgICAgICAgICAgb3BhY2l0eTogMTtcbiAgICAgICAgICB9XG4gICAgICAgICAgLnBhZ2UtdHJhbnNpdGlvbi1leGl0LWFjdGl2ZSB7XG4gICAgICAgICAgICBvcGFjaXR5OiAwO1xuICAgICAgICAgICAgdHJhbnNpdGlvbjogb3BhY2l0eSAke1RJTUVPVVR9bXM7XG4gICAgICAgICAgfVxuICAgICAgICAgIC5sb2FkaW5nLWluZGljYXRvci1hcHBlYXIsXG4gICAgICAgICAgLmxvYWRpbmctaW5kaWNhdG9yLWVudGVyIHtcbiAgICAgICAgICAgIG9wYWNpdHk6IDA7XG4gICAgICAgICAgfVxuICAgICAgICAgIC5sb2FkaW5nLWluZGljYXRvci1hcHBlYXItYWN0aXZlLFxuICAgICAgICAgIC5sb2FkaW5nLWluZGljYXRvci1lbnRlci1hY3RpdmUge1xuICAgICAgICAgICAgb3BhY2l0eTogMTtcbiAgICAgICAgICAgIHRyYW5zaXRpb246IG9wYWNpdHkgJHtUSU1FT1VUfW1zO1xuICAgICAgICAgIH1cbiAgICAgICAgYH08L3N0eWxlPlxuICAgICAgPC8+XG4gICAgKVxuICB9XG59XG4iXX0= */
| /*@ sourceURL=C:\Users\umuto\Desktop\with-next-page-transitions\pages\_app.js */`));
[ error ] { Error: Cannot find module 'C:\Users\umuto\Desktop\with-next-page-transitions\.next\server\static\development\pages\_document'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.loadComponents (C:\Users\umuto\Desktop\with-next-page-transitions\node_modules\next\dist\next-server\server\load-components.js:21:25)
    at DevServer.findPageComponents (C:\Users\umuto\Desktop\with-next-page-transitions\node_modules\next\dist\next-server\server\next-server.js:335:40)
    at DevServer.renderErrorToHTML (C:\Users\umuto\Desktop\with-next-page-transitions\node_modules\next\dist\next-server\server\next-server.js:466:35)
    at DevServer.renderErrorToHTML (C:\Users\umuto\Desktop\with-next-page-transitions\node_modules\next\dist\server\next-dev-server.js:17:1042)
    at process._tickCallback (internal/process/next_tick.js:68:7) code: 'MODULE_NOT_FOUND' }
{ Error: Cannot find module 'C:\Users\umuto\Desktop\with-next-page-transitions\.next\server\static\development\pages\_document'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.loadComponents (C:\Users\umuto\Desktop\with-next-page-transitions\node_modules\next\dist\next-server\server\load-components.js:21:25)
    at DevServer.findPageComponents (C:\Users\umuto\Desktop\with-next-page-transitions\node_modules\next\dist\next-server\server\next-server.js:335:40)
    at DevServer.renderErrorToHTML (C:\Users\umuto\Desktop\with-next-page-transitions\node_modules\next\dist\next-server\server\next-server.js:466:35)
    at DevServer.renderErrorToHTML (C:\Users\umuto\Desktop\with-next-page-transitions\node_modules\next\dist\server\next-dev-server.js:17:1175)
    at process._tickCallback (internal/process/next_tick.js:68:7) code: 'MODULE_NOT_FOUND' }
[ event ] disposing inactive page(s): /, /next/dist/pages/_error
[ error ] ./pages/_app.js 62:830
Module parse failed: Bad escape sequence in untagged template literal (62:830)
File was processed with these loaders:
 * ./node_modules/next/dist/build/webpack/loaders/next-babel-loader.js
You may need an additional loader to handle the result of these loaders.
|       id: "2096339533",
|       __self: this
>     }, `.page-transition-enter{opacity:0;-webkit-transform:translate3d(0,20px,0);-ms-transform:translate3d(0,20px,0);transform:translate3d(0,20px,0);}.page-transition-enter-active{opacity:1;-webkit-transform:translate3d(0,0,0);-ms-transform:translate3d(0,0,0);transform:translate3d(0,0,0);-webkit-transition:opacity ${TIMEOUT}ms,-webkit-transform ${TIMEOUT}ms;-webkit-transition:opacity ${TIMEOUT}ms,transform ${TIMEOUT}ms;transition:opacity ${TIMEOUT}ms,transform ${TIMEOUT}ms;}.page-transition-exit{opacity:1;}.page-transition-exit-active{opacity:0;-webkit-transition:opacity ${TIMEOUT}ms;transition:opacity ${TIMEOUT}ms;}.loading-indicator-appear,.loading-indicator-enter{opacity:0;}.loading-indicator-appear-active,.loading-indicator-enter-active{opacity:1;-webkit-transition:opacity ${TIMEOUT}ms;transition:opacity ${TIMEOUT}ms;}
| /*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkM6XFxVc2Vyc1xcdW11dG9cXERlc2t0b3BcXHdpdGgtbmV4dC1wYWdlLXRyYW5zaXRpb25zXFxwYWdlc1xcX2FwcC5qcyJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiQUFvQzJCLEFBR3VCLEFBSUEsQUFLQSxBQUdBLEFBS0EsQUFJQSxVQXBCd0IsQUFJSCxBQUtqQyxBQUdvRCxBQUtwRCxBQUlvRCxtR0FoQjBDLFNBSjlGLEFBWUEsQUFTQSxnU0FoQkEiLCJmaWxlIjoiQzpcXFVzZXJzXFx1bXV0b1xcRGVza3RvcFxcd2l0aC1uZXh0LXBhZ2UtdHJhbnNpdGlvbnNcXHBhZ2VzXFxfYXBwLmpzIiwic291cmNlc0NvbnRlbnQiOlsiaW1wb3J0IFJlYWN0IGZyb20gJ3JlYWN0J1xuaW1wb3J0IEFwcCBmcm9tICduZXh0L2FwcCdcbmltcG9ydCB7IFBhZ2VUcmFuc2l0aW9uIH0gZnJvbSAnbmV4dC1wYWdlLXRyYW5zaXRpb25zJ1xuXG5pbXBvcnQgTG9hZGVyIGZyb20gJy4uL2NvbXBvbmVudHMvTG9hZGVyJ1xuXG5jb25zdCBUSU1FT1VUID0gNDAwXG5cbmV4cG9ydCBkZWZhdWx0IGNsYXNzIE15QXBwIGV4dGVuZHMgQXBwIHtcbiAgc3RhdGljIGFzeW5jIGdldEluaXRpYWxQcm9wcyAoeyBDb21wb25lbnQsIGN0eCB9KSB7XG4gICAgbGV0IHBhZ2VQcm9wcyA9IHt9XG5cbiAgICBpZiAoQ29tcG9uZW50LmdldEluaXRpYWxQcm9wcykge1xuICAgICAgcGFnZVByb3BzID0gYXdhaXQgQ29tcG9uZW50LmdldEluaXRpYWxQcm9wcyhjdHgpXG4gICAgfVxuXG4gICAgcmV0dXJuIHsgcGFnZVByb3BzIH1cbiAgfVxuXG4gIHJlbmRlciAoKSB7XG4gICAgY29uc3QgeyBDb21wb25lbnQsIHBhZ2VQcm9wcyB9ID0gdGhpcy5wcm9wc1xuICAgIHJldHVybiAoXG4gICAgICA8PlxuICAgICAgICA8UGFnZVRyYW5zaXRpb25cbiAgICAgICAgICB0aW1lb3V0PXtUSU1FT1VUfVxuICAgICAgICAgIGNsYXNzTmFtZXM9J3BhZ2UtdHJhbnNpdGlvbidcbiAgICAgICAgICBsb2FkaW5nQ29tcG9uZW50PXs8TG9hZGVyIC8+fVxuICAgICAgICAgIGxvYWRpbmdEZWxheT17NTAwfVxuICAgICAgICAgIGxvYWRpbmdUaW1lb3V0PXt7XG4gICAgICAgICAgICBlbnRlcjogVElNRU9VVCxcbiAgICAgICAgICAgIGV4aXQ6IDBcbiAgICAgICAgICB9fVxuICAgICAgICAgIGxvYWRpbmdDbGFzc05hbWVzPSdsb2FkaW5nLWluZGljYXRvcidcbiAgICAgICAgPlxuICAgICAgICAgIDxDb21wb25lbnQgey4uLnBhZ2VQcm9wc30gLz5cbiAgICAgICAgPC9QYWdlVHJhbnNpdGlvbj5cbiAgICAgICAgPHN0eWxlIGpzeCBnbG9iYWw+e2BcbiAgICAgICAgICAucGFnZS10cmFuc2l0aW9uLWVudGVyIHtcbiAgICAgICAgICAgIG9wYWNpdHk6IDA7XG4gICAgICAgICAgICB0cmFuc2Zvcm06IHRyYW5zbGF0ZTNkKDAsIDIwcHgsIDApO1xuICAgICAgICAgIH1cbiAgICAgICAgICAucGFnZS10cmFuc2l0aW9uLWVudGVyLWFjdGl2ZSB7XG4gICAgICAgICAgICBvcGFjaXR5OiAxO1xuICAgICAgICAgICAgdHJhbnNmb3JtOiB0cmFuc2xhdGUzZCgwLCAwLCAwKTtcbiAgICAgICAgICAgIHRyYW5zaXRpb246IG9wYWNpdHkgJHtUSU1FT1VUfW1zLCB0cmFuc2Zvcm0gJHtUSU1FT1VUfW1zO1xuICAgICAgICAgIH1cbiAgICAgICAgICAucGFnZS10cmFuc2l0aW9uLWV4aXQge1xuICAgICAgICAgICAgb3BhY2l0eTogMTtcbiAgICAgICAgICB9XG4gICAgICAgICAgLnBhZ2UtdHJhbnNpdGlvbi1leGl0LWFjdGl2ZSB7XG4gICAgICAgICAgICBvcGFjaXR5OiAwO1xuICAgICAgICAgICAgdHJhbnNpdGlvbjogb3BhY2l0eSAke1RJTUVPVVR9bXM7XG4gICAgICAgICAgfVxuICAgICAgICAgIC5sb2FkaW5nLWluZGljYXRvci1hcHBlYXIsXG4gICAgICAgICAgLmxvYWRpbmctaW5kaWNhdG9yLWVudGVyIHtcbiAgICAgICAgICAgIG9wYWNpdHk6IDA7XG4gICAgICAgICAgfVxuICAgICAgICAgIC5sb2FkaW5nLWluZGljYXRvci1hcHBlYXItYWN0aXZlLFxuICAgICAgICAgIC5sb2FkaW5nLWluZGljYXRvci1lbnRlci1hY3RpdmUge1xuICAgICAgICAgICAgb3BhY2l0eTogMTtcbiAgICAgICAgICAgIHRyYW5zaXRpb246IG9wYWNpdHkgJHtUSU1FT1VUfW1zO1xuICAgICAgICAgIH1cbiAgICAgICAgYH08L3N0eWxlPlxuICAgICAgPC8+XG4gICAgKVxuICB9XG59XG4iXX0= */
| /*@ sourceURL=C:\Users\umuto\Desktop\with-next-page-transitions\pages\_app.js */`));
```

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

follow the instructons on this link: https://github.com/zeit/next.js/tree/canary/examples/with-next-page-transitions 

## Expected behavior

App should start

## System information

- OS: Windows10
- Version of Next.js: 9.1.1 (as latest)

**Comment by Tajwebdeveloper** [comment]
I have the exact same issue it seems to be happening only on "run dev" command "build" command works fine and the issue only happens on windows can't replicate the same on my mac. The workaround i found is to remove the jsx attribute from style tags and add before build.

**Comment by Tajwebdeveloper** [comment]
I guess nobody from next.js team uses the windows os so they are unaware about the bug

**Comment by umutozdemir97** [comment]
I realized I experience this error when I use style jsx tags with any dynamic feature on Windows

**Comment by Tajwebdeveloper** [comment]
> I realized I experience this error when I use style jsx tags with any dynamic value on Windows

yes exactly any updates?

**Comment by umutozdemir97** [comment]
There isn't any update on it yet

**Comment by emielta** [comment]
This is because your username is "unmuto" which results "\unmuto" (from the path) to be in the css string produced by the output of jsx (it's injected as `/*@ sourceURL=xxx */`) . This in turn results in a "Bad escape sequence in untagged template literal".

**Comment by emielta** [comment]
Should be fixed in b929debd76559640894978cae9c855793364cb92

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
